pkg/controller/secret/secret_controller.go

```diff
@@ -233,9 +259,8 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 		{Receiver: receiverNull, Match: map[string]string{"alertname": "PrometheusNotIngestingSamples", "namespace": "openshift-user-workload-monitoring"}},
 
 		//https://issues.redhat.com/browse/OSD-7671
+		// might also be removed by https://issues.redhat.com/browse/OSD-11273
 		{Receiver: receiverNull, Match: map[string]string{"alertname": "FluentdQueueLengthBurst", "namespace": "openshift-logging", "severity": "warning"}},
```

Comment:
this was not in the official alerts, not sure what version of alerts I need to pull this from?